cmd/kaspawallet/send.go

```diff
@@ -99,3 +99,15 @@ func send(conf *sendConfig) error {
 
 	return nil
 }
+
+/**
+ */
+func kasToSompi(amount float64) uint64 {
+	amountInStr := fmt.Sprintf("%.8f", amount)
+
+	parts := strings.Split(amountInStr, ".")
+
+	convertedAmount, _ := strconv.ParseUint(strings.Join(parts, ""), 10, 64)
```

Comment:
Check the error value (and return error is it is set). Number could be too big, and then ParseUint will return (nil, error)

Reply:
Addressed in [83c3d44](https://github.com/kaspanet/kaspad/pull/2220/commits/83c3d4458eb886ef2272130026d35ece56cbf946)